src/App.js

```diff
@@ -1,25 +1,25 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import AddItem from './components/AddItem';
+import List from './components/List';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import NavBar from './components/NavBar/NavBar';
```

Comment:
I'm seeing some inconsistencies in terms of folder/file naming and structure. For folders, we have lowercase (it could indicate either kebab-case or camelCase) `components` and PascalCase `NavBar`. 

For css files, I'm seeing a mix of camelCase and kebab-case. 

Some of the components don't have a dedicated folder, whereas NavBar does and then for the import we end up with `NavBar/NavBar`, which feels a little odd.

It may be a good idea to develop some conventions amongst the team in terms of file/folder naming conventions. Really, it's not super important what the convention is because that's really a stylistic choice (though I would argue that React component should always be PascalCase), but it is important that you develop, communicate, and stick to a given convention.

Reply:
Thank you very much for your feedback. We already did some changes. We'll bring it up to the rest of the team, so we can agree and stick to a naming and structure convention.